spec/views/sign_up/recovery_codes/show.html.slim_spec.rb

```diff
@@ -43,4 +21,30 @@
 
     expect(rendered).to have_css('.step-3.active')
   end
+
+  context 'when render has no dependencies' do
```

Comment:
I think having them all grouped makes total sense, but context is confusing as-written. I think the cost of render in view specs is pretty small so perhaps clearer to just render in each spec and do away with the context block